package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@actions/core": "^1.2.0",
     "@hapi/joi": "15.1.1",
+    "cli-table": "^0.3.1",
```

Comment:
Perhaps better to use a more managed package: https://www.npmjs.com/package/cli-table3

Reply:
Good one! I'll update that tomorrow.

Reply:
If you give me commit access to release-drafter/release-drafter I can contribute it as a co-maintainer 🚀